include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +63,67 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void append(Output& out, T const& v)
+    {
+        out << v;
+    }
+};
+
+template <typename CharT, typename Traits>
+struct output_formatter<std::basic_string<CharT, Traits>>
```

Comment:
`std::basic_string` has 3 template arguments prior to C++11. `std::basic_string` takes `Allocator` [1] while `std::basic_stringstream` does not [2].

[1] https://en.cppreference.com/w/cpp/string/basic_string
[2] https://en.cppreference.com/w/cpp/io/basic_stringstream

Reply:
Thankyou for the explanation.